wp-seo-main.php

```diff
@@ -260,9 +260,10 @@ function wpseo_init() {
  * Used to load the required files on the plugins_loaded hook, instead of immediately.
  */
 function wpseo_frontend_init() {
-	add_action( 'init', 'initialize_wpseo_front' );
+	WPSEO_Frontend::get_instance();
```

Comment:
This is exactly not what we want to do to make the tests pass ;-)This actually initializes the frontend even before `plugins_loaded`. Please revert that change and fix in the tests themselves.